docs/setup/self-hosted/docker.mdx

```diff
@@ -86,44 +68,63 @@ Now you can access the MindsDB editor by going to `127.0.0.1:47334` in your brow
   <img src="/assets/mindsdb-local-editor.png" />
 </p>
 
-## Install dependencies
+<Warning>
+If you experience any issues related to MKL or your training process does not complete, please add the `MKL_SERVICE_FORCE_INTEL` environment variable, as below.
+
+```bash
+docker run -e MKL_SERVICE_FORCE_INTEL=1 -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+```
+</Warning>
+
+### Install dependencies
 
 MindsDB integrates with numerous data sources and AI frameworks. To use any of the integrations, you should enure that the required dependencies are installed in the Docker container.
 
-Here is how to install the dependencies in the Docker container:
+<Info>
+Alternative way is to install dependencies directly from MindsDB editor. Go to *Settings* and *Manage Integrations*, select integrations you want to use and click on *Install*.
+</Info>
 
-1. Run the MindsDB Docker container:
-    ```bash
-    docker run -d -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
-    ```
+Run the MindsDB Docker container:
 
-2. Start an interactive shell in the container:
-    ```bash
-    docker exec -it container-name sh
-    ```
+```bash
+docker run -d -p 47334:47334 -p 47335:47335 mindsdb/mindsdb
+```
 
-3. Install the dependencies:
-    ```bash
-    pip install mindsdb[handler_name]
-    ```
-    For example, run this command to install dependencies for the [OpenAI handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/openai_handler):
-    ```bash
-    pip install mindsdb[openai]
-    ```
+Start an interactive shell in the container:
 
-4. Exit the interactive shell:
-    ```bash
-    exit
-    ```
+```bash
+docker exec -it container-name sh
+```
 
-5. Restart the container:
-    ```bash
-    docker restart container-name
-    ```
+Install the dependencies:
 
-## Configuration Options
+```bash
+pip install mindsdb[handler_name]
+```
+
+For example, run this command to install dependencies for the [OpenAI handler](https://github.com/mindsdb/mindsdb/tree/staging/mindsdb/integrations/handlers/openai_handler):
+
+```bash
+pip install mindsdb[openai]
+```
+
+Exit the interactive shell:
+
+```bash
+exit
+```
 
-### Default Configuration
+Restart the container:
+
+```bash
+docker restart container-name
```

Comment:
If we are going to include this, do you think it will be important to point out how the name of the container can be found? If the name is not specified in the `docker run` command, Docker automatically assigns a random name to the container, therefore, users will need to run a `docker ps` command to find out what the name assigned to their container is.

Reply:
I modified `docker run` and added info about `docker ps`.